userguide/one_shot_object_detection/README.md

```diff
@@ -0,0 +1,109 @@
+# One-Shot Object Detection
+
+One-Shot object detection is the task of learning to detect a set of categories from a handful of images. Unlike the Object Detector which requires many varied examples of objects in the real world, the One-Shot detector only requires a single canonical example of an object to train the classifier on and performs predictions for that category in the wild.
+
+Given an image and a one-shot detector trained on category 'stop sign', the output prediction will look like:
+
+![Stop Sign vector image](images/red_stop_sign.png) ![Stop Sign prediction image](images/stop_sign_prediction_sample.png)
+
+
+#### Introductory Example
+
+In this example, the goal is to localize where in the image instances of the category 'stop sign' are present. For this task we supply a single image of a stop sign as a starter image:
+
+
+```
```

Comment:
If you use `python` right after the starting three back ticks, you should get python syntax highlighting.